### DISCUSSION TITLE
Why do round pizzas come in square boxes?

### DISCUSSION THREAD

**Discussion by rerpha** [ISISComputingGroup/IBEX#7004] [Q&A] [answered]
Surely it's wasted space?

**Comment by rerpha** [comment]
FYI i'm just testing the search feature out with this one 😀

**Comment by JackEAllen** [comment, marked as answer]
According to [Pizza Planet](https://www.pizzaplanet.com/round-pizza-come-square-box/), square boxes are used simply because they are easier to manufacture from one piece of cardboard for a variety of different sizes. 👨‍🍳🍕

It's also apparently safer to have a square box to protect a pizza in transit.

However, in 1000AD, pizzas apparently used to be square which begs the question, have pizzas ever been triangles?
> 1000 AD in Italy and it is said that the first version was in a square shape. Ancestral versions looked more like a flat bread with meats. This ancient shape may have something to do with the way the box eventually evolved.

Never have I learned so much about why pizza boxes a square.
I feel enlightened...  😂
